src/components/Swap/Toolbar/index.tsx

```diff
@@ -79,6 +79,18 @@ export default memo(function Toolbar() {
     return inputBalance && inputAmount && inputBalance.lessThan(inputAmount)
   }, [inputAmount, inputBalance])
 
+  const maybeToggleOpen = useCallback(() => {
```

Comment:
Define this after the caption, taking advantage of the (new; see below) `isTogglable` boolean within the callback.